web-search-engines/langchain4j-web-search-engine-searchapi/src/main/java/dev/langchain4j/web/search/searchapi/SearchApiWebSearchEngine.java

```diff
@@ -0,0 +1,184 @@
+package dev.langchain4j.web.search.searchapi;
+
+import static dev.langchain4j.internal.Utils.getOrDefault;
+import static dev.langchain4j.internal.Utils.isNotNullOrBlank;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+import static java.time.Duration.ofSeconds;
+import static java.util.stream.Collectors.toList;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import dev.langchain4j.web.search.WebSearchEngine;
+import dev.langchain4j.web.search.WebSearchInformationResult;
+import dev.langchain4j.web.search.WebSearchOrganicResult;
+import dev.langchain4j.web.search.WebSearchRequest;
+import dev.langchain4j.web.search.WebSearchResults;
+import dev.langchain4j.web.search.searchapi.result.OrganicResult;
+import lombok.Builder;
+
+/**
+ * Represents SearchApi Search API as a {@code WebSearchEngine}. See more
+ * details <a href="https://www.searchapi.io/docs/google">here</a>.
+ */
+public class SearchApiWebSearchEngine implements WebSearchEngine {
+	public static final String DEFAULT_ENV_VAR = "SEARCHAPI_API_KEY";
```

Comment:
`DEFAULT_ENV_VAR` should not be here

Reply:
@langchain4j Should I move `DEFAULT_ENV_VAR` to a different class or remove the const completely?

Reply:
it is used only in integration test, so please move it there

Reply:
Okay. Moved to [`SearchApiWebSearchEngineIT`](https://github.com/ayewo/langchain4j/blob/49bbeb51c338a49ae1d1b35af6189aa445177345/web-search-engines/langchain4j-web-search-engine-searchapi/src/test/java/dev/langchain4j/web/search/searchapi/SearchApiWebSearchEngineIT.java#L21)